### PR TITLE
POLIO-926: campaign history logs some fields are not displayed

### PR DESCRIPTION
**Pull request by codeKameleon**
- budget log keys and values were missing 
- duplicated keys

Related JIRA tickets : [POLIO-926](https://bluesquare.atlassian.net/browse/POLIO-926?atlOrigin=eyJpIjoiYWViZTlhMzM5YzA1NDVkN2JhMWY0ZmQ1MDEyZGU2OWEiLCJwIjoiaiJ9)

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new string have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

-  add logic in `getLabel` to remove workflow suffix 'at_WFEDITABLE' for budget keys
- removed duplicated keys

## How to test

- Go to campaigns
- Choose a campaign
- Fill the dates in Budget tab
- Go to campaign log history and check if all fields are in the log as well as the values are well displayed and in good date format

[POLIO-926]: https://bluesquare.atlassian.net/browse/POLIO-926?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hakifran** [comment]
@codeKameleon while testing I noticed that two fields still missing in the history: Disbursed to MOH (WHO) and Disbursed to MOH (unicef)
Annotate-a-local-image(48)

File: plugins/polio/js/src/components/campaignHistory/config.tsx [hunk 1]
```diff
@@ -182,79 +182,79 @@ export const config: Record<string, any> = [
     },
     // Budget
     {
-        key: 'budget_status',
```
Comment (beygorghor) [inline plugins/polio/js/src/components/campaignHistory/config.tsx]: I guess those field needs to stay has previous campaigns can still use it no ? @olethanh  ?

**Review by beygorghor — CHANGES_REQUESTED** [review]

File: plugins/polio/js/src/components/campaignHistory/config.tsx (hunk 1 above)
Comment (olethanh) [reply plugins/polio/js/src/components/campaignHistory/config.tsx]: altough we could still check in db if it was actually ever used

Comment (codeKameleon) [reply plugins/polio/js/src/components/campaignHistory/config.tsx]: @beygorghor @olethanh I created an other ticket to add those fields : [POLIO-939](https://bluesquare.atlassian.net/browse/POLIO-939?atlOrigin=eyJpIjoiNWQ4OGU3ZDMyMzdhNGNlZWJiZWIxYWEzZmQxZjBiN2UiLCJwIjoiaiJ9)

**Comment by codeKameleon** [comment]
@cypress

**Review by beygorghor — APPROVED** [review]

**Pull request merged**